src/codegen.cpp

```
@@ -183,7 +183,7 @@ void CodeGen::emitTraceSymbolDefine(const std::string &kind, const std::string &
     // Skip stdlib symbols to reduce noise — users care about their own definitions
     if (sm_ && loc.file_id >= 0 && loc.file_id < sm_->getFileCount()) {
         const auto &fname = sm_->getFilename(loc.file_id);
-        if (fname.find("/lib/std/") != std::string::npos)
+        if (fname.find("/share/std/") != std::string::npos)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)